src/screens/ProfileScreen/index.js

```diff
@@ -273,6 +287,7 @@ const ProfileScreen = ({route}) => {
     if (interactionsComplete) {
       initialMyFeed();
       getProfileCache();
+      fetchFollower();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fetchFollower` function is called inside a `useEffect` hook when `interactionsComplete` is true. This seems fine as long as `interactionsComplete` is managed properly.
```diff
+  fetchFollower();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->